.github/pull_request_template.md

```diff
@@ -34,7 +34,7 @@ If this PR introduces user-facing changes, please note them here.
  - [ ] Add example if relevant.
  - [ ] Document any changes to public APIs.
  - [ ] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
- - [ ] Add an entry inside this element for inclusion in the `mapbox-maps-android` changelog: `<changelog></changelog>`.
+ - [ ] Add an entry inside [CHANGELOG.md](https://github.com/mapbox/mapbox-maps-android/blob/main/CHANGELOG.md) together with this PR, otherwise, apply the 'skip changelog' label.
```

Comment:
Hmm, could we remove it completely since CI will anyway fail the build if changelog wasn't modified?

Reply:
I guess would be good to add an item here before developer sees the error and fix? I think it's generally a work flow that developers who want to contribute should be aware of.